junebug/plugins/nginx/location.template

```diff
@@ -0,0 +1,3 @@
+location %(external_path)s {
+  proxy_pass %(internal_url)s;
+}
```

Comment:
Sometimes you set the headers `X-Real-IP`, `Host`, etc. But I don't think any of our transports care about those things; I think that this is okay.